src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -25,6 +26,7 @@
 from assisted_test_infra.test_infra.utils import logs_utils, network_utils, operators_utils
 from assisted_test_infra.test_infra.utils.waiting import wait_till_all_hosts_are_in_status
 from service_client import InventoryClient, log
+from tests.config import TerraformConfig
```

Comment:
Do not import `tests` package into `assisted_test_infra` package, you can use the abstract base configs object instead.

Reply:
But something is not clear to me here, why do we need to import the controller configurations in `cluster.py`? The cluster object have nodes object that managing the platform. 
Cluster supposed to not be depend on specific platform and therefore we don't need to import any controller configurations to `cluster.py`

Reply:
Gotcha! This was my attempt at trying to understand and make the code "work", thank you for the explanation! I've removed this import from `cluster.py`